ui/apps/platform/src/Containers/Policies/Table/PoliciesTablePage.tsx

```diff
@@ -167,7 +193,8 @@ function PoliciesTablePage({
 
     useEffect(() => {
         fetchPolicies(query);
-    }, [query]);
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, [query, sortOption]);
```

Comment:
Nit: you could update the call to `fetchPolicies` to require the sortOption as well, which would keep it consistent with the usage of `query` and allow us to remove this eslint-disable-next-line.

If I remember correctly, the object for `sortOption` is referentially equal while the options are unchanged.